src/queue.js

```diff
@@ -1,4 +1,21 @@
 class Queue {
+  constructor() {
+    this.queue = [];
+    this.size = this.queue.length;
+  }
+  enqueue(x) {
+    this.queue.unshift(x);
+    this.size = this.size + 1;
```

Comment:
Start getting in the habit of using the `+=` shorthand for operations like this. So you'd have `this.size += 1` here. You can do this with any math operation, such as `-=`, `*=`, `/=`, and even `%=`.